docs/documentation.md

```diff
@@ -17,6 +17,24 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+# RabbitMQ Documentation
+
+Welcome to RabbitMQ documentation. RabbitMQ is a reliable and mature messaging and streaming broker, which is easy to deploy on cloud environments, on-premises, and on your local machine. With support for many different messaging protocols, it can scale from providing point-to-point messaging all the way up to globally distributed highly-available event streaming with rock-solid reliability. RabbitMQ is currently used by millions worldwide. 
+
+## How to use this documentation
+
+Use the navigation on the left to browse through documentation for your release of RabbitMQ. This table of contents is structured so it can be used by our two main RabbitMQ personas: Developer and Administrator. 
```

Comment:
I think I preferred when the introduction was more to the point. That first paragraph is for the homepage. I mean, either the user is browsing the website to learn more about the project and probably started with the homapage, or they are here for the docs and would probably prefer to get the information they want straight away, not a general paragraph about something they are already using.